sidebarsDocs.js

```diff
@@ -93,6 +94,15 @@ module.exports = {
     },
     'api/console',
   ],
+  minercli: [
+    'mine-hnt/miner-cli/minercli',
```

Comment:
When we create a nested menu like you have, we usually make the link at the top a back button, just replace this line with below to do so.

```
{
      type: 'link',
      label: '<- Back',
      href: ''mine-hnt/miner-cli/minercli'
}
```